fr.esrf.tango.generator.xtend/src/fr/esrf/tango/pogo/generator/python/PythonDevice.xtend

```diff
@@ -194,6 +194,16 @@ class PythonDevice implements IGenerator {
             «IF attr.read»    self.attr_«attr.name»_read = «attr.defaultValueDim»«ENDIF»
         «ENDFOR»
             «cls.setEventCriteria»
+            «IF !cls.commands.empty»
+            «FOR cmd:cls.commands»
+            «IF cmd.name == "State" && cmd.polledPeriod.integerValue !== 0»
```

Comment:
[Minor] Can be simply to one IF:

```java
«IF (cmd.name == "State" || cmd.name == 'Status') && cmd.polledPeriod.integerValue !== 0»
self.poll_command("«cmd.name»", «cmd.polledPeriod»)
«ENDIF»
```

Reply:
Indeed, thanks. Changed into PythonDeviceHL then.